Mathlib/Algebra/Algebra/Operations.lean

```diff
@@ -216,13 +220,13 @@ theorem bot_mul : ⊥ * M = ⊥ :=
   map₂_bot_left _ _
 #align submodule.bot_mul Submodule.bot_mul
 
-@[simp]
+@[simp, nolint simpNF] -- Porting note: simp can't prove this
 protected theorem one_mul : (1 : Submodule R A) * M = M := by
```

Comment:
@jcommelin: can you explain this change to me? Surely simp can prove this via `_root_.one_mul`?